lib/saml_idp/xml_security.rb

```diff
@@ -298,6 +318,10 @@ def extract_inclusive_namespaces
         )&.attr('PrefixList')&.split(' ') || []
       end
 
+      def cert_element
+        @cert_elementy || document.at_xpath('//ds:X509Certificate | //X509Certificate', DS_NS)
```

Comment:
typo in the ivar name? and this looks like a memoization pattern?
```suggestion
        @cert_element ||= document.at_xpath('//ds:X509Certificate | //X509Certificate', DS_NS)
```

Reply:
yikes i am tired.

Reply:
updated in https://github.com/18F/saml_idp/pull/126/commits/1eb5cca1e7498f700556ecce5c72989fad66a218